security-proxy/src/main/java/org/georchestra/security/Proxy.java

```diff
@@ -956,7 +955,7 @@ private HttpRequestBase makeRequest(HttpServletRequest request, String sURL) thr
 
             // handles webdav specific verbs
             String[] webdavVerb = { "COPY", "LOCK", "UNLOCK", "MKCOL", "MOVE", "PROPFIND", "PROPPATCH", "UNLOCK",
-                    "REPORT" };
+                    "REPORT", "SEARCH" };
```

Comment:
As discussed during the conf-call, we are not sure if these are all entity/body based (as `POST` is) ; for sure the ones from the former PR were (`PROPFIND` & `SEARCH`).